src/js/controller/controller.js

```diff
@@ -576,6 +576,10 @@ Object.assign(Controller.prototype, {
             const adState = _getAdState();
 
             if (adState) {
+                // prevents the interaction with controls during ad playback to overwrite autostart playReason
+                if (_model.get('autostart') && adState === 'paused') {
+                    _model.set('playReason', 'autostart');
```

Comment:
What happens if I pause an ad and then click play again? Reason shouldn't be autostart if it's manually triggered.

Reply:
if the player is set to autostart: "true" in the initial setup, it will show "autostart" for playReason after you pause and play during the ad. If the player is set to autostart: "false", the play reason would show "interaction". I figured interacting with the ad playback should not overwrite the playReason for actual VOD playback but lmk if I'm mistaken!

Reply:
I believe you're mistaken. The reason should only be autostart for the initial play, as a play caused by what I'm describing above is triggered by an interaction.

Reply:
Find a way to pass `{ reason: 'autostart' }` when calling this function from the autostart scenario.